TheBTeam.Web/Views/Shared/_Layout.cshtml

```diff
@@ -3,15 +3,15 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - TheBTeam.Web</title>
+    <title>@ViewData["Title"] - Financial Planner</title>
```

Comment:
Yeah it only makes sense to change the title, we got a logo anyway

Reply:
[Display(Name = "Type)"] <= Here you have small error which don't a;;ow to compile project , fix it to
 [Display(Name = "Type")]